### PR TITLE
Rel 4.13.0.5

### PR DESCRIPTION
**Pull request by Sakoram**

### CODE REVIEW

**Comment by miczyg1** [comment]
@Sakoram tested on hardware?

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
 
+## [v4.13.0.5] - 2021-03-31
+### Changed
+- rebased with official coreboot repository commit e7a68ec
+
+### Added
+- [resource allocator v4 support](https://doc.coreboot.org/releases/coreboot-4.13-relnotes.html#resource-allocator-v4)
```
Comment (miczyg1) [inline CHANGELOG.md:21]: It is not merged: https://github.com/pcengines/coreboot/pull/465
Please remove

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Comment by macpijan** [comment]
@miczyg1 boot tested on apu1 and apu2.

File: CHANGELOG.md (hunk 1 above)
Comment (Sakoram) [reply CHANGELOG.md]: You right. Sorry, I was sure that this was merged. Removed.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**